charts/kueue/templates/prometheus/monitor.yaml

```diff
@@ -16,5 +16,5 @@ spec:
       insecureSkipVerify: true
   selector:
     matchLabels:
-      control-plane: controller-manager
+    {{- include "kueue.selectorLabels" . | nindent 6 }}
```

Comment:
Is this actually a fix? It seems that we were already doing `include "kueue.selectorLabels"` in other places where this PR removes `control-plane: controller-manager`?

Reply:
Yes, `{{- include "kueue.selectorLabels" . | nindent 6 }}` should have been there already.

Reply:
I see, could the fact it was not there cause some user-facing impact? I'm wondering if this fix should go in a separate PR, potentially as part of the 0.6 release (cherry-picked).